docs/configuration/monitoring.md

```diff
@@ -118,13 +118,13 @@ Now that you have Prometheus and Grafana installed you can enable `serviceMonito
     codacy-api:
       metrics:
         serviceMonitor:
-          enabled: true
```

Comment:
This dosent match the scentence above:

Now that you have Prometheus and Grafana installed you can enable `serviceMonitor`.

Maybe we should remove this for now

Reply:
agreed 👍

Reply:
I commented out the grafana documentation while our service monitors are not compatible with helm 3.